sink/sink

```diff
@@ -202,14 +206,14 @@ class GitHub(object):
         watches = (github.get("watches") or []) + (http.get("watches") or [])
         if watches:
             self.child = os.fork()
-            if self.child == 0:
+            if self.child > 0:
+                signal.signal(signal.SIGUSR1, github_usr1_handler)
```

Comment:
There's a race here.  The child might raise the signal before the parent has its handler setup.  Either use some form of synchronisation to avoid that, or set the handler up before the `fork()`.

In fact, it makes sense to set up the handler exactly once, at the start, and not at each `fork()`.

Reply:
I'll set the handler up before the fork :)